tests/BenchmarkDotNet.Tests/Loggers/DelegateLogger.cs

```diff
@@ -0,0 +1,28 @@
+﻿using BenchmarkDotNet.Loggers;
+using System;
+
+namespace BenchmarkDotNet.Tests.Loggers
+{
+    internal class DelegateLogger : ILogger
+    {
+        private readonly Action<LogKind, string> _writeAction;
```

Comment:
In BenchmarkDotNet, we don't use the underscore prefix for private fields. Please, rename it to `writeAction`.